### PR TITLE
Ability to disable rendering of welcome message

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
With `withWelcomeMessege` param

### CODE REVIEW

**Comment by wheresrhys** [comment]
can this be done as a map of properties following the same naming conventions as n-ui i.e `{{if @root.nUi.welcomeMessage}}`

**Comment by ironsidevsquincy** [comment]
Good idea

File: layout/wrapper.html [hunk 1]
```diff
@@ -87,7 +87,10 @@
 			<div class="n-layout__row n-layout__row--footer">
 				{{#outputBlock 'footer'}}{{/outputBlock}}
 				{{#if @root.flags.welcomePanel}}
-					{{>n-ui/welcome-message/template}}
+					{{#ifEquals nUi.welcomeMessage false}}
```
Comment (i-like-robots) [inline layout/wrapper.html]: Could use `{{#unless ... }}` maybe?

Comment (ironsidevsquincy) [inline layout/wrapper.html]: Unfortunately not, as that would mean updating every app that uses this template (as `undefined` would count as a positive)

**Pull request merged**